test/index.ts

```diff
@@ -0,0 +1,27 @@
+import 'mocha';
+import { startServer } from '../server';
+import axios from 'axios';
+import { expect } from 'chai';
+
+describe('Array', function () {
+  before('starting server', function (done) {
+    startServer().then(done);
+  });
```

Comment:
```suggestion
  before('starting server', async () {
    await startServer();
  });
```

before tb aceita async, a vantagem disso é que se tivesse outra etapa do setup do server (e vai ter) não iria ficar dentro do `then` (causando o callback hell que o Adalberto comentou no workshop de async/await)

Reply:
dei uma travada boa quando fui colocar essa etapa do setup do server no before, tentei muita coisa diferente (incluindo isso) mas toda vez ele acabava passando pro teste em si antes de finalizar o setup do server

mas acho que o que estava dando problema deveria ser alguma coisa na função startServer em si, pq realmente fazendo desse jeito dessa vez funcionou (enfim, add no commit seguinte)